modules/porous_flow/include/bcs/PorousFlowHalfCubicSink.h

```diff
@@ -0,0 +1,57 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+
+#ifndef POROUSFLOWHALFCUBICSINK_H
+#define POROUSFLOWHALFCUBICSINK_H
+
+#include "PorousFlowSink.h"
+
+// Forward Declarations
+class PorousFlowHalfCubicSink;
+
+template<>
+InputParameters validParams<PorousFlowHalfCubicSink>();
+
+/**
+ * Applies a flux sink to a boundary.  The base flux
+ * defined by PorousFlowSink is multiplied by a cubic.
+ * Denote x = porepressure - centre.  Then
```

Comment:
Here in 'merica, they spell `centre` as `center`. I do not really care too much, but if somebody feels strongly about consistent English throughout the code, `center` should be used... Not just here, but in the rest of the code as well.